Polyfills/Canvas/Source/Image.cpp

```diff
@@ -0,0 +1,77 @@
+#include <bgfx/bgfx.h>
+#include <map>
+#include "Canvas.h"
+#include "Image.h"
+#include "Context.h"
+#include <functional>
+#include <sstream>
+#include <assert.h>
+#include <NativeEngine.h>
+
+namespace Babylon::Polyfills::Internal
+{
+    static constexpr auto JS_CONSTRUCTOR_NAME = "NativeCanvasImage";
+
+    void NativeCanvasImage::CreateInstance(Napi::Env env)
+    {
+        Napi::HandleScope scope{env};
+
+        Napi::Function func = DefineClass(
```

Comment:
What about `crossOrigin`?

Reply:
I don't know yet what to do with `crossOrigin`

Reply:
That's fine. I noted it since you put everything else except for this.